examples/hogwart_objection/index.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const dotenv = require("dotenv");
+const url = require("url");
+const cors = require("cors");
+const { Hogwarts } = require("./model");
+const lib = require("@bitovi/querystring-parser");
```

Comment:
Once these packages are published, we'll want to swap this out with `@bitovi/objection-querystring-parser` right?
I manually tested this by pointing at `../../packages/objection`.
I'm getting errors back for filter query params. I think the objection (and maybe sequelize) packages need to be updated to handle the `#` prefixes in the column names.

<img width="1304" alt="image" src="https://user-images.githubusercontent.com/16355881/181313180-2f0c86c7-49f2-4f61-9457-806649858975.png">

Reply:
Yes, but since they haven't been published. I left it that way

Reply:
Right, yes I think it's fine to leave line 6 as is until the ORM packages are released.
But my real concern is that the objection and sequlize packages seem to be broken for `filter` parsing.